cloud/deploys/bin/azure/key-vault.sh

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env bash
+
+#######################################
+# Create resource group
+# Arguments:
+#   1: The resource group name for the key vault
+#   2: The location of the key vault
+#######################################
+function key_vault::create_resource_group(){
+    if [ $(az group exists --name "${1}") = false ]; then
+        az group create --name "${1}" -l "${2}"
```

Comment:
suggestion (blocking): use full/long flags for all if possible e.g. `--location`

Reply:
done